package.json

```diff
@@ -161,7 +161,7 @@
     "stoppable": "^1.1.0",
     "ts-toolbelt": "^9.6.0",
     "type-is": "^1.6.18",
-    "unleash-client": "3.21.0",
+    "unleash-client": "4.1.0-beta.3",
```

Comment:
support for strategy variants that will be used in frontend API but not playground which has its own impl